### ISSUE TITLE
GitHub Workflow for find_repeat_i18n_values.py

### ISSUE THREAD

**Issue by to-sta** [activist-org/activist#530]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

Task is to create a github workflow to run `find_repeat_i18n_values.py'.

### Contribution

Will work on this issue myself

**Comment by andrewtavis** [comment]
Glad my little script is getting a bigger role, @to-sta! 😊

**Comment by andrewtavis** [comment]
Should we rename is from `worker`, @to-sta?

**Comment by to-sta** [comment]
@andrewtavis I get rid of <del>`cronjob`</del> in the titel. 
Yeah renaming the workflow would make sense. Any ideas on how to rename this? (maybe i18n_check?)

Here my thoughts on improving the workflow/script:

I think there are two ways of improving this:
1. Raising an error in the script
2. Capturing the output of the script in the workflow

<ins>1. Raising an error in the script:</ins>
We can implement a logic in the script to raise an error based on the repeat values. 

<ins>2. Capturing the output of the script in the workflow:</ins>
This would allows to create an Issue/PR message with the output. The downside of this is that we potential create a lot of issues if we have consecutive commits.

**Comment by andrewtavis** [comment]
Have it renamed locally now to `i18n_value_check`, @to-sta :) Let me send along the most recent changes shortly and we can talk about it further. I decided to alter the script so that it will let us know what the keys of the repeat values are and also suggest the lowest level of key we can use for the renamed key-value pair 😊 We could potentially capture the output and send it to the PR as a message. Making a bunch of good first issues would be nice, but then we need to make sure that repeats aren't being merged :)

**Comment by andrewtavis** [comment]
40b7944 and fb04997 update the workflow and [find_repeat_i18n_values.py](https://github.com/activist-org/activist/blob/main/frontend/i18n/find_repeat_i18n_values.py). It's not perfect, but will give us good suggestions and also lists the keys that need to be find all replaced throughout. General workflow I'm envisioning is find all replace to a common key that will then create a key conflict in the JSON, and then just delete one of the two keys 😊

**Comment by andrewtavis** [comment]
How are you feeling on the final steps of operationalizing it? Do you want to write something up that sends a message to the PR. We can also simply raise a value error at the end, which should fail it and we'd then be able to check the workflow to see the printed statements? I'd say this is fine enough as this is what we're doing with the rest of the workflows - see fail and check the workflow itself.

**Comment by to-sta** [comment]
I can add something for the PR, but it will not work on push. Let's raise an error that is simplier and coveres both cases.

**Comment by andrewtavis** [comment]
Sounds good :) Just 39 left now! 😅

**Comment by andrewtavis** [comment]
Hey @to-sta 👋 b912388 adds a script and a workflow step for a step that was missing in all this: checking to see if we have keys that aren't being used anywhere for cases where keys are being renamed/reassigned and we then have one sitting there that doesn't do anything 😊 As we have two steps, I also renamed again to `pr_ci_i18n` to match the others :)

Let me know what you think! There are a few final edits of keys and values that are needed, but I'm hoping to have this done by tomorrow 🚀

**Comment by to-sta** [comment]
That's a great addition to this workflow 🚀!

I was wondering why the workflow was not triggered on `push`. I made a small error in the workflow:

```yaml
on:
  ...
  push:
    branches: -main  # <- This should be on the next line - main
    paths:
      - "**/en-US.json"
      
  # Like this
  push:
    branches: 
      - main
    paths:
      - "**/en-US.json"
```

**Comment by andrewtavis** [comment]
Fixed in 353059c, @to-sta 😊

**Comment by andrewtavis** [comment]
Checks have finished, @to-sta! See the [commit with all the i18n changes](https://github.com/activist-org/activist/actions/runs/6788929657/job/18454987120). I renamed `worker` to `i18n_check` in the file to finish this :) :) Thanks for all the work here!
